### DISCUSSION TITLE
Dependency Management

### DISCUSSION THREAD

**Discussion by fxmb** [fal-ai/dbt-fal#199] [Q&A]
We would like to use Fal for Feature Engineering whenever SQL is not strong enough. How would we manage dependencies here, e.g. have pandas etc installed but not globally but simply for the specific script we are running? Ideally we would do this in a docker env?

Any pointers? 🙏

**Comment by chamini2** [comment]
Hello, @franzbewerunge; thanks for reaching out!

I think the Docker approach would work fine. I would include a `requirements.txt` with the necessary dependencies and install them in the docker environment.

Are you expecting very different dependencies in your scripts that would need separate environments for each? If so, maybe we could explore your use case in detail and see what we can come up with!

**EDIT**: We have a ticket about dependency management https://github.com/fal-ai/fal/issues/10 that we haven't tackled yet.
